core/commands/dag/import.go

```diff
@@ -2,15 +2,15 @@ package dagcmd
 
 import (
 	"errors"
-	"fmt"
 	"io"
 
 	cid "github.com/ipfs/go-cid"
 	ipld "github.com/ipfs/go-ipld-format"
 	ipldlegacy "github.com/ipfs/go-ipld-legacy"
 	"github.com/ipfs/go-libipfs/files"
-	iface "github.com/ipfs/interface-go-ipfs-core"
 	"github.com/ipfs/interface-go-ipfs-core/options"
+	"github.com/ipfs/interface-go-ipfs-core/path"
+
```

Comment:
```suggestion
```
We don't do that, this might be a usefull style but then we should add a a formater that does this for us automatically.